src/main/java/org/folio/service/pieces/flows/update/PieceUpdateFlowInventoryManager.java

```diff
@@ -127,6 +128,8 @@ private Future<String> handleItem(PieceUpdateHolder holder, RequestContext reque
   private Future<Void> updateItemWithFields(JsonObject item, CompositePoLine compPOL, Piece piece) {
     Optional.ofNullable(piece.getHoldingId())
       .ifPresent(pieceHoldingId -> item.put(ITEM_HOLDINGS_RECORD_ID, piece.getHoldingId()));
+    Optional.ofNullable(piece.getCopyNumber())
+      .ifPresentOrElse(enumeration -> item.put(COPY_NUMBER, enumeration), () -> item.remove(COPY_NUMBER));
```

Comment:
why we need to remove copy number here?

Reply:
It's added becouse we can clean 'copyNumber' field in the piece and if it's happened we clean the same field in item.